hax/pmt_plot.py

```diff
@@ -0,0 +1,154 @@
+##
+## Plot some data on the PMT arrays
+##
+## Jelle, February 2016
+## May go into hax soon...
+##
+## Known issue's I'm to lazy to fix right now:
+##  - on physical layout, color and/or size probably not on same scale in two subplots 
+##    unless vmin&vmax are specified explicitly.
+##  - Have categorical labels event if _channel present. Make digitizer obey _channel suffix convention.
```

Comment:
__[flake8]__
* [E266] too many leading '#' for block comment
* [E501] line too long (104 > 79 characters)